app/views/shared/_metadata.html.erb

```diff
@@ -0,0 +1,16 @@
+<% p = @monograph_presenter || @presenter %>
+<% if p.present? && ( p.model_name.singular == 'file_set' || p.model_name.singular == 'monograph' ) %>
+  <meta name="citation_title" content="<%= p.page_title %>">
+  <meta name="citation_author" content="<%= p.creator_full_name %>">
+  <% p.contributor.each do |contributor| %>
+  <meta name="citation_author" content="<%= contributor %>">
+  <% end %>
```

Comment:
Sorry I didn't mention this yesterday -- but can we also add identifiers?

If a DOI exists:

`<meta name="citation_doi" content="10.3998/fulcrum.123134">`

Otherwise, use the handle:

`<meta name="DC.identifier" content="2027/fulcrum.123234">`

Note that the `https://doi.org/` or `http://hdl.handle.net/` parts are stripped out of the `content` attribute.